python/lsst/pipe/tasks/measurePsf.py

```diff
@@ -278,12 +277,15 @@ def run(self, exposure, sources, expId=0, matches=None):
         reserveList = []
 
         if self.config.reserveFraction > 0:
-            random.seed(self.config.reserveSeed*expId)
-            reserveList = random.sample(psfCandidateList,
-                                        int((self.config.reserveFraction)*len(psfCandidateList)))
-
-            for cand in reserveList:
-                psfCandidateList.remove(cand)
+            random = afwMath.Random(seed = self.config.reserveSeed*expId)
+            reserveList = []
```

Comment:
See https://developer.lsst.io/coding/python_style_guide.html#keyword-assignment-operators-should-be-surrounded-by-a-space-when-statements-appear-on-multiple-lines
> However, if keyword assignments occur on a single line, where should be no additional spaces

(@jonathansick, typo: where ==> there)